libinstall/CentWeb.sh

```diff
@@ -587,11 +621,6 @@ $INSTALL_DIR/cinstall $cinstall_opts \
 check_result $? "$(gettext "Install centreonSyncArchives")"
 
 ## Install generateSqlLite
-log "INFO" "$(gettext "Prepare generateSqlLite")"
-cp $TMP_DIR/src/bin/generateSqlLite \
-    $TMP_DIR/final/bin/generateSqlLite >> "$LOG_FILE" 2>&1
```

Comment:
Are you sure the files are copied into final directory ? (Needed just after)

Reply:
yes they are.
from the temporary src (original file) to the final